client/verta/tests/test_model_registry/test_model_version.py

```diff
@@ -462,3 +463,27 @@ def test_patch(self, registered_model):
         assert version.get_labels() == LABELS
 
         assert version.get_attributes() == ATTRIBUTES
+
+# TODO: move related tests from TestModelVersion before merging
```

Comment:
Why hasn't it been done?

Reply:
It would've made the diff a lot more verbose for the reviewer.

Reply:
Cool